001-hello-pipecat-nim.ipynb

```diff
@@ -10,127 +10,94 @@
     "\n",
     "Pipecat AI is an open-source framework for building voice and multimodal conversational agents. Pipecat simplifies the complex voice-to-voice AI pipeline, and lets developers build AI capabilities easily and with Open Source, commercial, and custom models. See [Pipecat's Core Concepts](https://docs.pipecat.ai/getting-started/core-concepts) for a deep dive into how it works.\n",
     "\n",
-    "The framework was developed by Daily, a company that has provided real-time video and audio communication infrastructure since 2016. It is fully vendor neutral and is not tightly coupled to Daily's infrastructure. That said, we do use it in this demo. Sign up for a Daily-bots API key [here](https://bots.daily.co/sign-up)."
+    "The framework was developed by Daily, a company that has provided real-time video and audio communication infrastructure since 2016. It is fully vendor neutral and is not tightly coupled to Daily's infrastructure. That said, we do use it in this demo. Sign up for a Daily-bots API key [here](https://bots.daily.co/sign-up).\n",
+    "\n",
+    "Below is the architecture diagram:\n",
+    "\n",
+    "![Architecture Diagram](./arch.png)\n",
+    "\n",
+    "A three-phase approach is used for Conversational AI Agent with Pipecat and NVIDIA NIM:\n",
+    "\n",
+    "#### Phase 1 : User Input\n",
+    "- Audio Processing with NVIDIA RIVA ASR with NIM\n",
+    "\n",
+    "#### Phase 2: User Content Aggregator with Pipecat and NVIDIA NIM\n",
+    "- Custom processing with Pipecat\n",
+    "- NVIDIA RIVA TTS with NIM\n",
+    "\n",
+    "#### Phase 3: Run the Agent\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3ceb5b93",
+   "metadata": {},
+   "source": [
+    "# Content Overview \n",
+    "\n",
+    "- [Prerequisites](#prerequisites)\n",
+    "- [Initialize the User Input](#initialize-the-user-input)\n",
+    "- [Initialize the Content Aggragtor](#initialize-the-context-aggregator) \n",
+    "- [Run the Agent](#run-the-agent)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "9b4fa7d7-88fb-4b33-8145-ee1a91e58af1",
    "metadata": {},
    "source": [
-    "## Step 1 - Install dependencies\n",
+    "## Prerequisites\n",
+    "\n",
+    "### NGC API Key\n",
+    "Prior to getting started, you will need to create API Keys for the NVIDIA API Catalog, Tavily, and Langchain.\n",
```

Comment:
I don't recall creating API keys for Tavily or Langchain... are those needed for this blueprint?